README.md

```diff
@@ -69,3 +75,78 @@ When the Django server is running, you can start the Django shell with:
 Run the back-end test suite with:
 
     docker-compose exec backend pipenv run python manage.py test --settings=donate.settings_test
+
+
+## Basket
+
+[Basket](https://github.com/mozmeao/basket) is a tool run by MoCo to manage newsletter subscriptions and donations. It's listening for messages (JSON) sent to a SQS queue.
+
+### Basket donations queue:
+
+Basket has 4 [donation event types](https://github.com/mozmeao/basket/blob/master/basket/news/management/commands/process_donations_queue.py#L21-L26): 
```

Comment:
Are these more like "4 event types the foundation uses"?

Reply:
I would go for "Basket has 4 event types we can use".

Reply:
```suggestion
Basket has [4 event types we can use](https://github.com/mozmeao/basket/blob/master/basket/news/management/commands/process_donations_queue.py#L21-L26): 
```